oi/type_graph/Printer.cpp

```diff
@@ -132,6 +132,14 @@ void Printer::visit(const DummyAllocator& d) {
   print(d.allocType());
 }
 
+void Printer::visit(const CycleBreaker& b) {
+  if (prefix(&b))
+    return;
+
+  out_ << "CycleBreaker" << std::endl;
```

Comment:
If we're going to generate a name for CycleBreaker nodes then we should print it out as well - Printer is meant to be able to show every detail about a type graph. I do plan to move the NameGen tests over to the Printer-based testing framework at some point too.